cmd/manager/profileparser.go

```diff
@@ -0,0 +1,310 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	"github.com/operator-framework/operator-sdk/pkg/log/zap"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/subchen/go-xmldom"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	cmpv1alpha1 "github.com/openshift/compliance-operator/pkg/apis/compliance/v1alpha1"
+	"github.com/openshift/compliance-operator/pkg/profileparser"
+	"github.com/openshift/compliance-operator/pkg/xccdf"
+)
+
+var profileparserCmd = &cobra.Command{
+	Use:   "profileparser",
+	Short: "Runs the profile parser",
+	Long:  `The profileparser reads a data stream file and generates profile objects from it.`,
+	Run:   runProfileParser,
+}
+
+func init() {
+	defineRerunnerFlags(profileparserCmd)
+}
+
+// XMLDocument is a wrapper that keeps the interface XML-parser-agnostic
+type XMLDocument struct {
```

Comment:
It looks like we already have this type in the ARF parser, can we reuse it instead of duplicating?